currencies/rate_converter.go

```diff
@@ -70,7 +70,17 @@ func NewRateConverterWithNotifier(
 		return rc
 	}
 
-	rc.Update()                   // Make sure to populate data before to return the converter
+	// Make sure to populate data before to return the converter
+	err := rc.Update()
+	// If unable to fetch currency rates from "syncSourceURL" then log the error
+	// and move on without supporting currency conversions.
+	if err != nil {
+		glog.Errorf("Error fetching currency rates from %s. Currency conversions won't be supported. %s",
+			syncSourceURL, err.Error())
+		rc.constantRates = NewConstantRates()
+		return rc
```

Comment:
If the intent is to allow PBS to startup successfully when the rate server is unavailable, I think we should continue to try and hit the rates server during periodic updates for when the server is once again reachable. In that case, the error message makes sense but I would like to see the `startPeriodicFetching` routine still running. Ideally IMHO, Rates would return constant rates when no rate refresh should ever be attempted or when all previous attempts have failed.

Reply:
That's a good point ! +1 to let the routine try to fetch currency rates !

Reply:
@syntaxnode Yep, I like that approach better. Will update the PR. Thanks!